modules/Book.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+
+export const formBtn = document.querySelector("#form-button");
+export const booksList = document.querySelector(".books-list");
+export const books = JSON.parse(localStorage.getItem("books")) || [];
+
+export class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
+    const { id, title, author } = this;
+    const bookObj = { id, title, author };
+    if (books !== null) {
+      books.push(bookObj);
+      localStorage.setItem("books", JSON.stringify(books));
+      document.getElementById("title").value = "";
+      document.getElementById("author").value = "";
+    }
+  }
+
+  removeBook() {
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id !== id) {
+        return true;
+      }
+      return false;
+    });
+    localStorage.setItem("books", JSON.stringify(books));
+  }
+}
```

Comment:
- I have this error when I try to remove a book from the list, Please fix this 🙏 
![thdddh](https://user-images.githubusercontent.com/73607512/212747127-bac2d9d6-2db2-4081-9697-815bdb82992b.PNG)

Reply:
Thanks for noticing this bug. I would fix it.